src/main/java/seedu/address/model/person/Person.java

```diff
@@ -112,6 +129,19 @@ public boolean isSamePerson(Person otherPerson) {
                 && (otherPerson.getPhone().equals(getPhone()) || otherPerson.getEmail().equals(getEmail()));
     }
 
+    /**
+     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * This defines a weaker notion of equality between two persons.
+     */
```

Comment:
Do edit the comment above, i guess it's supposed to be both persons of the same employeeId